### PR TITLE
conditions unneeded components for arm

### PR DESCRIPTION
**Pull request by ar3ndt**

### CODE REVIEW

File: core/src/epicli/data/common/ansible/playbooks/roles/applications/tasks/applications/ignite-stateless/main.yml [hunk 1]
```diff
@@ -3,3 +3,4 @@
   include_tasks: deploy-k8s-app.yml
   vars:
     app_dir_name: "{{ data.name }}"
+  when: ansible_architecture != "aarch64"
```
Comment (rpudlowski93) [inline core/src/epicli/data/common/ansible/playbooks/roles/applications/tasks/applications/ignite-stateless/main.yml]: \n

File: core/src/epicli/data/common/ansible/playbooks/roles/applications/tasks/applications/istio/main.yml [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-
-- name: Deploy Istio
-  include_tasks: deploy-k8s-app.yml
-  vars:
-    app_dir_name: "{{ data.name }}"
+  - name: Deploy Istio
+    include_tasks: deploy-k8s-app.yml
+    vars:
+      app_dir_name: "{{ data.name }}"
+when: ansible_architecture != "aarch64"
```
Comment (rpudlowski93) [inline core/src/epicli/data/common/ansible/playbooks/roles/applications/tasks/applications/istio/main.yml]: \n

File: core/src/epicli/data/common/ansible/playbooks/roles/applications/tasks/applications/pgbouncer/main.yml [hunk 3]
```diff
@@ -3,3 +3,4 @@
   include_tasks: deploy-k8s-app.yml
   vars:
     app_dir_name: "{{ data.name }}"
+  when: ansible_architecture != "aarch64"
```
Comment (rpudlowski93) [inline core/src/epicli/data/common/ansible/playbooks/roles/applications/tasks/applications/pgbouncer/main.yml]: \n

File: core/src/epicli/data/common/ansible/playbooks/roles/applications/tasks/applications/pgpool/main.yml [hunk 4]
```diff
@@ -3,3 +3,4 @@
   include_tasks: deploy-k8s-app.yml
   vars:
     app_dir_name: "{{ data.name }}"
+  when: ansible_architecture != "aarch64"
```
Comment (rpudlowski93) [inline core/src/epicli/data/common/ansible/playbooks/roles/applications/tasks/applications/pgpool/main.yml]: \n

File: core/src/epicli/data/common/ansible/playbooks/zookeeper.yml [hunk 5]
```diff
@@ -9,4 +9,5 @@
   become: true
   become_method: sudo
   roles:
-    - zookeeper
+    - role: zookeeper
+      when: ansible_architecture != "aarch64"
```
Comment (rpudlowski93) [inline core/src/epicli/data/common/ansible/playbooks/zookeeper.yml]: \n

Comment (rpudlowski93) [reply core/src/epicli/data/common/ansible/playbooks/zookeeper.yml]: We want to have kafka on ARM but without zookeeper?

Comment (ar3ndt) [reply core/src/epicli/data/common/ansible/playbooks/zookeeper.yml]: zookeeper is needed, good point, Ill revert it back

**Comment by seriva** [comment]
This should be branched from [ARM](https://github.com/epiphany-platform/epiphany/tree/ARM) and merged back into ARM. Not from develop.

**Comment by ar3ndt** [comment]
wrong branch

**Pull request closed without merging**